day_6/exercises/burrito.js

```diff
@@ -20,8 +20,14 @@ class Burrito {
 
 var dinner = new Burrito("Beans", "Rice", ["cheese", "salsa", "guacamole"]);
 
-// Call the methods here
+dinner.toppings.push("avacado");
+dinner.toppings.shift();
+dinner.protein = "Mole";
```

Comment:
Good work reassigning Burrito attributes with built in js methods.

How would your `removeTopping` method change if you wanted to remove a specific topping?